SciTools Status Page

LukeC92 opened the pull request:
The main change from this request is the creation of a page that shows the build status and test coverage of any SciTools or SciTools-incubator packages we are interested in. It is not linked to from any other page but it does have the links to the rest of the site; this was done by adding build_status.html to build.py. I also added some things that may be helpful for this, if not now then maybe in the future; e.g. project.mobirise, datatables and some css formating.

I also added an example web page that I could use to copy code from, but I removed this on the last commit.

pelson commented:
This is the link for the docs: https://rawgit.com/LukeC92/scitools.org.uk/status_page/build_status.html
Unfortunately, it doesn't look right. @LukeC92, could you take a look please?

pelson commented on build_status.html:
```diff
@@ -0,0 +1,353 @@
+              </tr>
+            </thead>
+            <tbody>
+
```
Perhaps put a html comment in here to highlight that this is populated via JS at the end of the file

pelson commented on build_status.html:
```diff
@@ -0,0 +1,353 @@
+                     return $('<tr>'
+                            + ' <td class="body-item mbr-fonts-style display-7">' + repo.name + '</td>'
+                            + ' <td class="body-item mbr-fonts-style display-7">'
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
```
This is the wrong link.

pelson commented on build_status.html:
```diff
@@ -0,0 +1,353 @@
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
+                            + '     <img src="https://coveralls.io/repos/github/' + repo.org + '/' + repo.name + '/badge.svg?branch=master" alt="Iris Coverage Status" />'
+                            + '   </a>'
+                            + ' </td>'
```
Could you add the other badges like the ones at https://github.com/scitools/cf_units.

LukeC92 commented:
I think I have addressed all of the issues previously brought up. Though it's worth noting that having "?branch=master" on the end of many of the links doesn't seem to make a difference. It seems to make a difference to most, if not all, of the images. But the hrefs seem to go to the same place regardless of the branch specified. Hence I have remove these parts of the URLs. 

It may be that this makes a difference for some of the links and others just need a different format.

pelson commented:
This is fabulous @LukeC92! It is so useful!!! I'm super happy to have this 😄 🎉 🥇

pelson commented:
@SciTools/developers - this really neat page just landed: take a look at https://scitools.org.uk/build_status.html

We haven't integrated it into the scitools.org.uk pages yet, but those with the secret-sauce (the url... that isn't that secret) can see it. I for one will be checking this around once a week to ensure everything remains tickety-boo on our projects. It also highlights that I need to get back to fixing those pesky tests... 😜

lbdreyer commented:
@LukeC92 I suspect it is due to the shields badges not correctly understanding how to handle release candidates, so that it thinks v2.0.0rc1 is > v2.0.0 (See this comment: https://github.com/badges/shields/issues/490#issuecomment-205550621)

We could potentially raise it with the shields developers
